.github/workflows/pull_request.yml

```diff
@@ -9,13 +9,14 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - uses: extractions/setup-just@v1
-      - uses: actions/setup-python@v1
+      - uses: actions/setup-python@v2
         with:
-          python-version: 3.8
-      - name: Install Dependencies
-        run: pip install pre-commit
-      - name: Run static checks
-        run: just lint
+          python-version: 3.9
+      - uses: pre-commit/action@v2.0.3
+        env:
+          SKIP: isort  # Fixme: this isn't working on GHA, "setuptools" not found
```

Comment:
<img width="605" alt="Screen Shot 2022-01-03 at 3 50 59 PM" src="https://user-images.githubusercontent.com/13648427/147992494-bc7a2c54-7b93-4822-96a1-42c7e8622660.png">
FYI near as I can tell the isort test works just fine in GHAs. Not exactly sure what the issue was previously.

Reply:
Which run are you getting this from?

Reply:
Welp, guess whatever cache issue was happening is fixed now :upside_down_face: